common/kvstore/src/main/java/org/apache/spark/util/kvstore/LevelDBIterator.java

```diff
@@ -197,7 +197,6 @@ public synchronized void close() throws IOException {
    * when Scala wrappers are used, this makes sure that, hopefully, the JNI resources held by
    * the iterator will eventually be released.
    */
-  @SuppressWarnings("deprecation")
```

Comment:
@srowen I found that you added this `@SuppressWarnings` in SPARK-26090 . It seems that this is to suppress the `deprecation` warning when Java 8 -> 11.  So we don't need to delete it?

Reply:
Oh, do warnings pop back up? I thought these were found to not suppress anything? if it still emits a warning without it, we should keep it. For example, yeah, isn't finalize() definitely deprecated as of 11?

Reply:
Revert the change related Java 11, and others will not pop new warnings.